In [1]:
import pandas as pd
import numpy as np
import plotly.express as px
import scipy.stats as stats

In [6]:
df1 = pd.read_csv('혼자+생활하면서+곤란한+점_20240226134402.csv')

In [7]:
df1.columns

Index(['특성별(1)', '특성별(2)', '2020', '2020.1', '2020.2', '2020.3', '2020.4',
       '2020.5', '2020.6', '2020.7', '2020.8', '2020.9', '2020.10', '2020.11',
       '2020.12', '2020.13', '2020.14', '2020.15', '2020.16', '2020.17',
       '2020.18', '2020.19', '2020.20', '2020.21', '2020.22', '2020.23',
       '2020.24', '2020.25', '2020.26', '2020.27', '2020.28', '2020.29'],
      dtype='object')

In [13]:
df1['2020']

0         1순위
1     사례수 (명)
2        1009
3          53
4         297
5         126
6         352
7         181
8         392
9         617
10        315
11        302
12        242
13        150
14         78
15        553
16        323
17         36
18         11
19          7
Name: 2020, dtype: object

## 혼자 생활하면서 곤란한 점
- 전체(1009명)
- 남성:392, 여성:617/20대이하:315, 30-40대:302, 50~60대:242, 70대 이상: 150)
### 1순위
1. 몸이 아프거나 위급할 때 대처의 어려움(33%)
2. 외로움(23%)
3. 경제적 불안감(20%)
4. 강력범죄 등 안전에 대한 불안감(7%)
5. 혼자사는 사람에 대한 주위 시선(7%)
6. 임종에 대한 불안감(고독사의 문제)(4%)
7. 주거관리(댁내 장비 교체 등)의 어려움 (4%)
8. 지역사회 내 믿고 의지할 사람(이웃 주민의 부족)(3%)
9. 기타(1%)


## 혼자 생활하는 가장 큰 이유
- 전체(1009명)
- 남성:392, 여성:617/20대이하:315, 30-40대:302, 50~60대:242, 70대 이상: 150)

1. 본인의 직장 또는 학교와의 거리 때문에(49%)
2. 배우자와의 이혼 별거 사별 때문에(31%)
3. 개인적 편의(가족 및 주변으로부터의 사생활 보장 등)와 자유를 위해(10%)
4. 함께 살던 가족구성원의 이민(기러기가족 포함) 이사 또는 독립 (6%)
5. 함께 살던 가족과의 불화때문에(3%)
6. 부모 또는 형제자매의 사망 때문에(1%)
